migrate/v0_16/testdata/genesis-v16.json

```diff
@@ -1479,6 +1479,51 @@
         }
       ]
     },
+    "capability": {
```

Comment:
@karzak fixed the e2e genesis test by adding the default gen state for the ibc modules